docs/development.md

```diff
@@ -1,46 +1,85 @@
 # Development Setup
 
-## Docker
-
-Install docker and docker-compose.
+## Prerequisites
+
+* [Docker](https://docs.docker.com/get-started/)
+* [Docker Compose](https://docs.docker.com/compose/install/) (does not need to be installed separately if using Docker Desktop)
+* Python 3.9
+  * Optional: [pyenv](https://github.com/pyenv/pyenv) for managing Python versions
+* Node.js >= 20
+  * Optional: [nvm](https://github.com/nvm-sh/nvm) for managing Node.js versions
+* Yarn 1.x
+  * Recommended: install via `corepack`:
+    ```shell
+    corepack enable
+    corepack install --global yarn@1
```

Comment:
Side note: Interesting! I was not familiar with corepack until seeing it here.

I've been running yarn _via npx_ (i.e. `$ npx yarn`) any time the instructions have said to run `yarn`. I don't currently have `yarn`, itself, installed directly on my machine.

![image](https://github.com/user-attachments/assets/14c1a169-83a6-4555-bbb5-059c713c7ae0)

I'll look into using [corepack](https://github.com/nodejs/corepack) from now on.